src/blocks.tsx

```diff
@@ -182,8 +182,15 @@ export const InformationBlock: React.FC<InformationBlockProps> = ({
 interface HighlightedValueBlockProps {
     tag?: RendersNullableElement<typeof Tag>;
 
-    value: string | Array<string>;
-    text: string | Array<string>;
+    mainHeading: {
+        value: string;
+        text: string;
```

Comment:
this should be optional, right?

Reply:
The mainHeading, I believe it is not optional